src/generated/resources/data/forge/tags/blocks/ores/dense.json

```diff
@@ -0,0 +1,8 @@
+{
+  "replace": false,
+  "values": [
+    "minecraft:copper_ore",
+    "minecraft:lapis_ore",
+    "minecraft:redstone_ore"
```

Comment:
If the purpose of this tag is to identify all ores that have a 100% chance of more than 1 drop, then is there a reason why deepslate variants aren't included?

Reply:
Jup, you are correct 🤦‍♂️